### PR TITLE
Tracking settings

### PR DESCRIPTION
**Pull request by david8611**
This is the first attempt at an implementation of the tracking settings by @SamuelHarner and me. I decided to use a loop to create each switch and icon view adding an ID to each info-button that can be retrieved when adding an info popup when pressing the info button. The switch documentation can be found here: https://reactnative.dev/docs/switch and here: https://callstack.github.io/react-native-paper/switch.html. It inherits all the style properties from a View component. The Android implementation looks like this: 
Screenshot_20210222-024952 <width=50% height=50%>
And:
Screenshot_20210222-025000 width=50% height=50%

The web version looks like this:
Skärmklipp

I haven't yet been able to test the IOS version.
For some reason I also haven't been able to rezise the images in this comment.

### CODE REVIEW

**Comment by jscarlsson** [comment]
What's the best way to implement translations on this? Are the names (in the consts) important / used by the system, or are they just for display? Is there a reason 'CervicalMucus' doesn't have a space?

The point is it should use the i18n system. I merged it just now so it makes sense you didn't include it.

If the names are just for display, I think we can just change them the same as every other string, but I don't know it it becomes strange since they're in consts. The other way to do it would be to have some (simple) function that converts the names to actual strings for display.

Basically I think I'm just getting confused because they kind of look like enums, but I don't think that's what they are.

Either way, I can help with the i18n stuff if you want, but we'd need to do it together I think because I don't know how much would need to be changed here. Maybe just a little, maybe a bit more.

**Comment by david8611** [comment]
> What's the best way to implement translations on this? Are the names (in the consts) important / used by the system, or are they just for display? Is there a reason 'CervicalMucus' doesn't have a space?
> 
> The point is it should use the i18n system. I merged it just now so it makes sense you didn't include it.
> 
> If the names are just for display, I think we can just change them the same as every other string, but I don't know it it becomes strange since they're in consts. The other way to do it would be to have some (simple) function that converts the names to actual strings for display.
> 
> Basically I think I'm just getting confused because they kind of look like enums, but I don't think that's what they are.
> 
> Either way, I can help with the i18n stuff if you want, but we'd need to do it together I think because I don't know how much would need to be changed here. Maybe just a little, maybe a bit more.

No worries, I found a good solution to this. Basically I just add each translation in the for loop using array syntax, matching each const word in the corresponding arrays. An example can be found here: https://stackoverflow.com/questions/44007617/how-can-i-translate-an-array-with-react-native-i18n. The implementation can be found in the most recent commit.

**Review by SamuelHarner — APPROVED** [review]
Tracking settings screen has now been implemented as per design mock-up. Some adjustments for FAB positioning have also been added, which seems to get correct behavior on Android and in iOS simulator at least.
